docs/update_and_migration/from_4.5/update_from_4.5.md

```diff
@@ -6,6 +6,17 @@ description: Update your installation to the latest v4.6 version from v4.5.x.
 
 This update procedure applies if you are using a v4.5 installation.
 
+!!! note "Remove temporary Composer conflicts"
+
+    You can now remove the temporary Composer conflicts from your `composer.json` file:
+    ```diff
+        "conflict": {
+    -        "jms/serializer": ">=3.30.0",
+    -        "gedmo/doctrine-extensions": ">=3.12.0"
+        },
+    ```
+
+
 ## Update from v4.5.x to v4.5.latest
```

Comment:
```suggestion
## Update from v4.5.x to v4.5.latest
```

I feel like this isn't at the right place. This note about removing `conflict` should be in the update to v4.5.latest procedure's section if the fix is present in 4.5.